.s2i/bin/assemble

```diff
@@ -1,71 +0,0 @@
-#!/bin/bash -e
```

Comment:
@kquinn1204 This file deletion was in the other PR of yours that I just peer reviewed. I don't think it's supposed to be here. Can you remove it from this PR before we merge?

Reply:
Yeah I can see that your commit message changed to something about the .gitignore file. So if you do have to redo the commit, I'd recommend fixing the commit message here too.

Let me know if you need any help with removing that accidental file deletion that is showing up.